### ISSUE TITLE
KYC is wrong

### ISSUE THREAD

**Issue by EgorKuts** [Fantom-foundation/tokensale#10]
Dear colleagues, I expect justice from your organization. I trusted and waited for success. I have honestly passed all your conditions. I have made all possible procedures faultlessly. I waited when technical problems end, I spent time waiting for lotteries, and would accept any destiny to a lottery. But the lottery has passed. And you have published me as the winner. Be responsible for the words. You want that we were responsible for the words. The community expects the same from you. I consider that you is groundless have excluded me from the whitelist. You have no right without explanations to leave this situation. It looks so: you use our energy, belief and forces, and further you change the game course in own favor and you say to people no, only because you have an internal check, on yours to nobody to the known rules. It is sure that I have to be included in whitelist by your rules! I ask you to make a round of additional verifications of my account. If you have made a choice not in my advantage on the basis of a photo, then let me make to you 100 photos. If on the basis of the address, then I give 100 proofs of the address. If you hadn't liked my purse, then I will send you 100 facts that you didn't specify in rules what it has to be anywhere. I wait on at least constructive answer as a maximum of participation in your project. I believe in human justice. dima.hodak@bk.ru and eth= 0xbae50c7Ec0eF535BF049F23cE29Fd337a18D4F6C

**Comment by chhotesarkar** [comment]
I'm also faced the same problem

**Comment by BramCoding** [comment]
I have the exact same problem. KYC-checks passed with these exact documents many times on other cryptocurrency related sites. Now all of a sudden it doesn't pass? I messaged ALL admins and community managers, none have given me an answer to why my KYC didn't pass.

Ridiculous.

**Comment by EgorKuts** [comment]
yes, but they promised to give a chance to  resubmit form if something with docs goes wrong during kyc. They lied to get more hamsters and HYIP
